lib/system/threads.nim

```diff
@@ -177,18 +177,28 @@ else:
     else:
       type Time = int
 
+  when defined(linux) and defined(amd64):
+    type
+      SysThread* {.importc: "pthread_t", header: "<sys/types.h>" .} = distinct culong
+      Pthread_attr {.importc: "pthread_attr_t",
+                       header: "<sys/types.h>", final, pure.} = object
+        abi: array[56, uint8]
```

Comment:
I'm sure Pthread_attr needs alignment beyond uint8, so you need to use ``array[7, float64]`` or something.